src/otx/algorithms/detection/adapters/mmdet/task.py

```diff
@@ -513,6 +503,12 @@ def _export_model(
         assert len(self._precision) == 1
         export_options["precision"] = str(self._precision[0])
         export_options["type"] = str(export_format)
+        if self.max_num_detections > 0:
+            logger.info(f"Export max_num_detections: {self.max_num_detections}")
+            post_proc_cfg = export_options["deploy_cfg"]["codebase_config"]["post_processing"]
+            post_proc_cfg["max_output_boxes_per_class"] = self.max_num_detections
+            post_proc_cfg["keep_top_k"] = self.max_num_detections
+            post_proc_cfg["pre_top_k"] = self.max_num_detections * 10
```

Comment:
I'm not sure this may change postprocessing of exported model. How did you verify this?

Reply:
I've looked into ModelAPI & MMDeploy source code. The former had no such filtering, but MMDeploy was using those parameters for OpenVINO NMS layer parameters.
I've tested train -> export -> OV eval to verify if this approach worked.
Do you have any good idea for us to make this a test case?

Reply:
It's hard to test by unit test because it requires exported openvino.xml. Anyway, I just want to check whether this really changes behaviors of mmdeploy. Sometimes mmdeply ignores parameters in config file. If your check this e2e way then I think it's fine at the current stage.